code/modules/europa/clothing/helmet.dm

```diff
@@ -10,6 +10,19 @@
 		slot_l_hand_str = 'icons/mob/europa/lefthand_helmet.dmi',
 		slot_r_hand_str = 'icons/mob/europa/righthand_helmet.dmi'
 		)
+		
+/obj/item/clothing/head/europa
+	name = "Europan hat"
```

Comment:
yeah, should probably have been `"spaceproof Europan hat"` and `"\improper Europan hat"`, but eh